src/Packages/Microsoft.Internal.Extensions.DotNetApiDocs.Transport/Microsoft.Internal.Extensions.DotNetApiDocs.Transport.proj

```diff
@@ -0,0 +1,41 @@
+<Project Sdk="Microsoft.Build.NoTargets">
+
+    <PropertyGroup>
+        <TargetFramework>$(LatestTargetFramework)</TargetFramework>
+        <!-- Workaround https://github.com/microsoft/MSBuildSdks/issues/469 -->
+        <TargetFrameworks />
+        <IsShipping>false</IsShipping>
+        <!-- Reference the outputs to have them available as build outputs. -->
+        <NoTargetsDoNotReferenceOutputAssemblies>false</NoTargetsDoNotReferenceOutputAssemblies>
+        <IsPackable>true</IsPackable>
+        <IncludeBuildOutput>true</IncludeBuildOutput>
+        <IncludeSymbols>false</IncludeSymbols>
+        <EnablePackageValidation>false</EnablePackageValidation>
+        <SuppressDependenciesWhenPacking>true</SuppressDependenciesWhenPacking>
+        <PackageDescription>Internal transport package to provide dotnet-api-docs with the reference assemblies and compiler generated documentation files from dotnet/extensions.</PackageDescription>
+        <NoWarn>$(NoWarn);NU5128;NU5131</NoWarn>
+        <BuildProjectReferences Condition="'$(NoBuild)' == 'true'">false</BuildProjectReferences>
+        <TargetsForTfmSpecificContentInPackage>$(TargetsForTfmSpecificContentInPackage);IncludeProjectReferenceCompilerGeneratedSecondaryOutputInPackage</TargetsForTfmSpecificContentInPackage>
+    </PropertyGroup>
+
+    <ItemGroup>
+        <!-- Set PrivateAssets to avoid dependencies in the nupkg
+             Set Private=false to avoid copying the referenced projects to output -->
+        <ProjectReference Include="$(LibrariesProjectRoot)\*\*.*proj;$(LibrariesProjectRoot)\..\ToBeMoved\*\*.*proj;$(LibrariesProjectRoot)\..\ToBeRemoved\*\*.*proj" PrivateAssets="all" Private="false" />
```

Comment:
Please add a new line after each `;` charater. That makes it easier to read the items.

Reply:
I totally missed this before. I believe we only need to include `src\**\*.*proj`. As the names suggest `ToBeMoved` and `ToBeRemoved` aren't here for long :)

Reply:
When Carlos and I were trying this we noticed that the projects in ToBeMoved and ToBeRemoved would be included transitively by others, but would be missing the `Private="false"` marking and be copied into the output folder unnecessarily.  This was more of just a placeholder to spark discussion -- should transitive projects be included in this transport package if they weren't directly referenced?  Do folks care about copying the references into the output folder of Microsoft.Internal.Extensions.DotNetApiDocs.Transport?

Reply:
We don't care about the output folder of the transport package, it's more or less just an intermediate. We care about the package layout though. We require all shipping reference assemblies (or the implementation assembly, if dotnet/extensions doesn't ship reference assemblies) and their documentation files to be present.

Btw, @carlossanlop please make sure that the transport package doesn't include non-shipping intermediate reference assemblies that never reach customers. You might be doing that accidentally by using the `ReferenceAssembly` item metadata from the ProjectReferences.

Reply:
> When Carlos and I were trying this we noticed that the projects in ToBeMoved and ToBeRemoved would be included transitively by others, but would be missing the `Private="false"` marking and be copied into the output folder unnecessarily. This was more of just a placeholder to spark discussion -- should transitive projects be included in this transport package if they weren't directly referenced? Do folks care about copying the references into the output folder of Microsoft.Internal.Extensions.DotNetApiDocs.Transport?

That makes sense. I suppose the build will fail once we removed both of these folders, and we'll just adjust the config.
/resolved